HIP-5.md

```diff
@@ -0,0 +1,53 @@
+| hip | title     | status | author | created    |
+| --- | --------- | ------ | ------ | ---------- |
+| 5   | Combo ISM | Draft  | asaj   | 2022-12-21 |
+
+### **Brief Summary / Abstract**
+
+Defines the specification for an interchain security module (ISM) that combines other ISMs.
+
+### **Motivation**
+
+The goal of this HIP is to standardize the `ComboIsm` interface, so that users can compose multiple ISMs.
```

Comment:
Obviously again take any naming comment from me with a grain of salt, but when I hear `ComboISM`, it would imply the RoutingISM more than this one. I wonder if `MultiISM` would be a good option if it wasn't for the existence of the `MultisigISM`. `PolyIsm`? `ConsensusISM`? `ConcurrentISM`? `UnanimousISM`?

Reply:
Hmm, I wonder if others feel that way?

> Combination: a joining or merging of different parts or qualities in which the component elements are individually distinct.

Reply:
AggregationIsm it is!